assets/template-collection.css

```diff
@@ -349,35 +349,62 @@ button.facets__button {
   grid-row: 2;
 }
 
-a.active-facets__button {
+.active-facets__button {
+  padding: 1.1rem 0.5rem;
+  text-decoration: none;
+}
+
+.active-facets__button:first-child {
+  margin-left: 0.5rem
+}
+
+span.active-facets__button-inner {
   color: var(--color-foreground);
   box-shadow: 0 0 0 0.1rem var(--color-foreground);
   border-radius: 2.6rem;
-  font-size: 1.2rem;
+  font-size: 1rem;
```

Comment:
In terms of accessibility, I dont think font sizes are supposed to be smaller than 12px 🤔 

Source: https://accessibility.psu.edu/fontsizehtml/#:~:text=For%20traditional%20computer%20monitors%2C%20a,can%20be%20zoomed%20to%20200%25.

Reply:
Spoke to @Oliviammarcello and we've decided to keep it at `10px`.  The user can just pinch to zoom and make the size bigger if they need to.  This is easy to do on mobile.